### ISSUE TITLE
Audit READMEs and other docs for public repos

### ISSUE THREAD

**Issue by blackbaud-johnly** [blackbaud/skyux2-docs#887]
Got an internal request to make sure we are keeping up with READMEs and other docs that are publicly viewable in our open source docs to make sure they aren't out of date.

**Comment by blackbaud-johnly** [comment]
I went through the repos listed in the packages matrix at https://developer.blackbaud.com/skyux/contribute/packages. Most didn't have any content except for badges that link to npm and coverage charts. Here's what I found:

a11y: Nothing but badges.
action-bars: Nothing but badges.
ag-grid: Nothing but badges.
angular-tree-component: Nothing but badges.
animation: Nothing but badges.
assets: Nothing but badges.
auth-client-factory: Just a one-sentence description of the methods.
autonumeric: Badges plus basic instructions. I think the content is now represented in the docs or should be and that we should confirm that and then delete the content here.
avatar: Nothing but badges.
colorpicker: Nothing but badges.
config: Nothing but badges.
core: Nothing but badges.
datetime: Nothing but badges.
docs-tools: Installation instructions documented here. Should this be somewhere else or is this the best place?
errors: Nothing but badges.
flyout: Nothing but badges.
grids: Nothing but badges.
http: Nothing but badges.
i18n: Nothing but badges.
indicators: Nothing but badges.
inline-form: Nothing but badges.
layout: Nothing but badges.
list-builder: Nothing but badges.
list-builder-common: Nothing but badges.
list-builder-view-checklist: Nothing but badges.
list-builder-view grids: Nothing but badges.
lists: Nothing but badges.
lookup: Nothing but badges.
modals: Nothing but badges.
omnibar-interop: Nothing but badges.
phone-field: Nothing but badges.
popovers: Nothing but badges.
progress-indicator: Nothing but badges.
router: Nothing but badges.
select-field: Nothing but badges.
split-view: Nothing but badges.
tabs: Nothing but badges.
theme: Nothing but badges.
tiles: Nothing but badges.
toast: Nothing but badges.
validation: Nothing but badges.

builder: Nothing but badges.
builder-plugin-migrate: Nothing but badges.
builder-plugin-skyux: Badges, plus installation instructions. Is this the right place for those?
cli: Badges, plus content linking to the SKY UX docs site. Do we need this additional content here?
e2e: Badgets, plus a code sample for running visual tests.
pact: Nothing but badges.
testing: Nothing but badges.
builder-config: No badgets, but configuration info and pipeline settings. Should this be documented somewhere else?
template: No badges, but a short description.
template-library: No badges, but a short description.
template-stache: No badges, but a short description.

auth-client: Badges, plus installation and usage instructions. Should this be documented elsewhere?
help-client: Badges, plus configuration info and link to report issues. We are deprecating the widget, so we might should replace these docs with a note about the deprecation.
skyux2: Badges, plus overview and contribution info. Do we still need anything documented here?
skyux-builder: Badges, plus short description. Should we replace the description with a deprecation note and a link to skyux-sdk/builder?
skyux-builder-plugin-auth-email-whitelist: Badges, plus installation and usage info. Should this be documented elsewhere?
skyux-builder-plugin-stache: Badges, plus basic docs. Should this just link to the Stache docs?
skyux-builder-stache-search: Badges, plus basic docs. Should this just link to the Stache docs?
skyux-cli: Deprecation note, badges, and basic docs. Should we use this deprecation note elsewhere? And should we remove the rest of the docs?
skyux-deploy: Badges, plus basic docs. Should this be documented elsewhere?
skyux-design-tokens:  Badges, plus basic docs. Should this be documented elsewhere?
skyux-lib-clipboard: Badges, plus basic docs. Should this just link to the Stache docs?
skyux-lib-code-block: Badges, plus basic docs. Should this just link to the Stache docs?
skyux-lib-help: Basic docs. We are deprecating the widget, so we might should replace these docs with a note about the deprecation.
skyux-lib-media: Nothing but badges.
skyux-lib-restricted-view: Badges, plus short description and link to Stache docs.
skyux-lib-stache: Deprecation note, badges, and basic docs. Should we remove the docs except for the deprecation note?
skyux-lib-testing: Deprecation note, badges, and basic docs. Should we remove the docs except for the deprecation note?
skyux-logger: Just a basic description.
skyux2-docs: Just a code snippet and link for filing issues.


Not sure what's worth updating here. None of the content jumped out at me as problematic from a security standpoint, which I think was the concern that prompted this issue being filed.
